runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -4699,6 +4699,13 @@ J9::Z::TreeEvaluator::multianewArrayEvaluator(TR::Node * node, TR::CodeGenerator
    TR::LabelSymbol *cFlowRegionEnd = generateLabelSymbol(cg);
    TR::LabelSymbol *oolFailLabel = generateLabelSymbol(cg);
 
+#if defined(TR_TARGET_64BIT)
+   bool isIndexableDataAddrPresent = TR::Compiler->om.isIndexableDataAddrPresent();
+   TR::LabelSymbol *populateFirstDimDataAddrSlot = NULL;
```

Comment:
Can we use conditional assignment here ?

Reply:
Do you mean conditional assignment for `populateFirstDimDataAddrSlot`?

Reply:
Yes